### PR TITLE
Issue and Pull Request templates

### PR DESCRIPTION
**Pull request by mriehema**
- Add templates for github issues and pull requests. 
  See https://github.com/blog/2111-issue-and-pull-request-templates
- Add last pull request to CHANGELOG.md

### CODE REVIEW

**Comment by mriehema** [comment]
@eirslett What do you think about releasing a new version? I already changed CHANGELOG.md for a release in this PR. ;)

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<!-- Before creating an issue please make sure you are using the latest version of frontend-maven-plugin. -->
```
Comment (eirslett) [inline .github/ISSUE_TEMPLATE.md]: People should also search in the issue/PR archive to check whether anybody else has already requested the same.

**Comment by eirslett** [comment]
:+1:

File: .github/ISSUE_TEMPLATE.md (hunk 1 above)
Comment (mriehema) [inline .github/ISSUE_TEMPLATE.md]: I will change it

**Pull request merged**